File: noobs/q8/solution.py
def count_zero(arr):
    count = 0
    for i in arr:
        if(i == 0):
            count +=1
            continue
        
        while(i>0):
            if(i%10 == 0):
                count +=1
            i//=10
    return count

arr = [0,101010203040,0]
print (count_zero(arr))
        
            

File: noobs/q8/solution.cpp
#include<iostream>
#include<vector>
using namespace std;

int count_zero(vector<int> arr){
    int count = 0;

    for(int i : arr){
        if(i == 0){
            count++;
            continue;
        }

        while(i > 0){
            if(i % 10 == 0){
                count++;
            }
            i /= 10;   // <-- always divide
        }
    }

    return count;
}

int main(){
    vector<int> arr = {0,1010,3,30303303};
    cout << count_zero(arr) << endl;

    return 0;
}
